### PR TITLE
APP-627: Change docs to reflect system toolbar changes

### PR DESCRIPTION
**Pull request by kyliedale**
- Correctly reference new system toolbar changes
- Update screenshots to match the latest system designer
- Delete unused images/gifs

One thing left I would like to update is the breadcrumb gif. @jennyknuth I could use your help on that.

### CODE REVIEW

File: docs/ui/build-a-ui.md [hunk 1]
```diff
@@ -34,10 +34,10 @@ Follow these steps to create a simple UI that can publish to, subscribe to, and
 1. In the root of the project, rename **config.js.example** to `config.js`, and open it in a text editor.
     1. **Never commit configuration details! We've added `config.js` to `.gitignore`. Even though these are public urls and tokens, it's bad practice to commit them. For deployment, make sure you put `config.js` at your project root.**
 1. Get your Pubkeeper **hostname** and **token** from your nio-managed cloud-instance:
+    <img class="right border" src="/img/cloud/editSystem.png" width="250" />
     1. Open the nio **System Designer** in a browser: https://app.n.io/design.
-    1. Select your system in the left-hand navigation.
         1. If you need to create your first system, follow the instructions here: [docs.n.io/system-designer/designer-tasks.html#create-a-system](/system-designer/designer-tasks.md#create-a-system).
-    1. Click the **edit** button in the contextual toolbar to open its configuration.
+    1. Hover over the system card and lick the **edit** button in the toolbar that appears to open its configuration.
```
Comment (jennyknuth) [inline docs/ui/build-a-ui.md]: 'lick' haha!

File: docs/system-designer/designer-tasks.md [hunk 2]
```diff
@@ -206,12 +202,11 @@ In the **block library** search box, enter a block type name.
...
 
 1. Drag the block type to the canvas.
-1. In the **block name** box, enter a meaningful name and click **accept**.
-<img class="left border" src="/img/cloud/Hello-CreateBlockSimulate.png" width="250" />
+1. In the **name** box, enter a meaningful name and click **accept**.
 
 > **[info] Block names**
 >
-> Block names cannot be edited after creation.
+> Block names cannot be edited after creation in versions less than 3.0 of nio.
```
Comment (jennyknuth) [inline docs/system-designer/designer-tasks.md]: let's switch this around to start "In nio versions less than 3.0, block names cannot be edited after creation."

File: docs/running-nio/in-the-cloud.md [hunk 3]
```diff
@@ -30,26 +30,29 @@ A system is a group of interrelated elements. In the nio System Designer, a syst
 
   > **[info] <span class="allow-caps">Pubkeeper</span> server credentials**
   >
+  > <img class="right border" src="/img/cloud/editSystem.png" width="250" />
   > To access your **Pubkeeper** credentials:
-  > 1. Click on your system's name on the leftmost side of the System Designer.
-  > 1. Click the **edit** button in the contextual toolbar above the canvas.
+  > 1. Click on the **design** app icon in the app switcher on the left side of the System Designer.
+  > 1. Hover over the system card that you wish to get the Pubkeeper credentials of.
+  > 1. Click the **edit** button on the card.
```
Comment (jennyknuth) [inline docs/running-nio/in-the-cloud.md]: Excellent info box! What do you think about switching the order in step one so it is place -> action? "In the app switcher on the left side of the System Designer, click the **design** app icon." Step two ends with an awkward preposition, but I am not sure how best to fix: "Hover over the card of the system whose Pubkeeper credentials you need." ? Actually, I really like this sentence from above, "You can see the configuration of the cloud Pubkeeper server if you hover over a system card and click the **edit** icon in the toolbar." but does that fit here in these steps?

File: docs/system-designer/designer-tasks.md [hunk 4]
```diff
@@ -54,28 +51,27 @@ Instances are created inside a system and contain services.
 
 ### Create an instance in the cloud
 
-To enter the system context, select the name of your system in the left navigation panel or on the breadcrumb above the contextual toolbar.
+To enter the system context, select the system card from the grid or click on the breadcrumb.
```
Comment (jennyknuth) [inline docs/system-designer/designer-tasks.md]: no "click on", just use "click"

File: docs/system-designer/designer-tasks.md [hunk 5]
```diff
@@ -54,28 +51,27 @@ Instances are created inside a system and contain services.
...
+1. Check the **cloud instance** checkbox.
+1. Complete the **add instance** modal:
   * In the **instance name** box, enter a meaningful name.
 1. Click **accept**.
 
-
 ### Connect to a local instance
 
-To enter the system context, select the name of your system in the left navigation panel or on the breadcrumb above the contextual toolbar.
+To enter the system context, select the system card from the grid or click on the breadcrumb.
```
Comment (jennyknuth) [inline docs/system-designer/designer-tasks.md]: remove "on"

File: docs/running-nio/in-the-cloud.md (hunk 3 above)
Comment (kyliedale) [reply docs/running-nio/in-the-cloud.md]: I like it as the last step. Changing now.

**Pull request merged**